### ISSUE TITLE
Need help with ceph. Today all PVC unmounted and cant be create/mount again.

### ISSUE THREAD

**Issue by mixMugz** [ceph/ceph-csi#2547]
# Describe the bug #
Today our dev product stop working. All PVC cant mount, new cant create (stay in Pending state)
No any changes in CEPH/Kubernetes clusters.

There is a log from csi-provisioner

```
I0929 17:13:10.289585       1 controller.go:1317] provision "default/csi-cephfs-pvc" class "cephfs": started
I0929 17:13:10.289791       1 controller.go:573] CreateVolumeRequest {Name:pvc-d373ab9b-1497-4718-b9f4-783d1ef5a5d4 CapacityRange:required_bytes:1073741824  VolumeCapabilities:[mount:<mount_flags:"debug" > access_mode:<mode:MULTI_NODE_MULTI_WRITER > ] Parameters:map[clusterID:d26925a3-b527-4eb4-9b3a-23625d055104 csi.storage.k8s.io/controller-expand-secret-name:csi-cephfs-secret csi.storage.k8s.io/controller-expand-secret-namespace:ceph-csi-cephfs csi.storage.k8s.io/node-stage-secret-name:csi-cephfs-secret csi.storage.k8s.io/node-stage-secret-namespace:ceph-csi-cephfs csi.storage.k8s.io/provisioner-secret-name:csi-cephfs-secret csi.storage.k8s.io/provisioner-secret-namespace:ceph-csi-cephfs fsName:cephfs pool:cephfs_data] Secrets:map[] VolumeContentSource:<nil> AccessibilityRequirements:<nil> XXX_NoUnkeyedLiteral:{} XXX_unrecognized:[] XXX_sizecache:0}
I0929 17:13:10.290027       1 event.go:282] Event(v1.ObjectReference{Kind:"PersistentVolumeClaim", Namespace:"default", Name:"csi-cephfs-pvc", UID:"d373ab9b-1497-4718-b9f4-783d1ef5a5d4", APIVersion:"v1", ResourceVersion:"43895367", FieldPath:""}): type: 'Normal' reason: 'Provisioning' External provisioner is provisioning volume for claim "default/csi-cephfs-pvc"
I0929 17:13:10.296946       1 connection.go:182] GRPC call: /csi.v1.Controller/CreateVolume
I0929 17:13:10.296994       1 connection.go:183] GRPC request: {"capacity_range":{"required_bytes":1073741824},"name":"pvc-d373ab9b-1497-4718-b9f4-783d1ef5a5d4","parameters":{"clusterID":"d26925a3-b527-4eb4-9b3a-23625d055104","fsName":"cephfs","pool":"cephfs_data"},"secrets":"***stripped***","volume_capabilities":[{"AccessType":{"Mount":{"mount_flags":["debug"]}},"access_mode":{"mode":5}}]}
I0929 17:13:10.313760       1 connection.go:185] GRPC response: {}
I0929 17:13:10.313821       1 connection.go:186] GRPC error: rpc error: code = InvalidArgument desc = failed to get connection: connecting failed: rados: ret=-13, Permission denied
I0929 17:13:10.313862       1 controller.go:645] CreateVolume failed, supports topology = false, node selected false => may reschedule = false => state = Finished: rpc error: code = InvalidArgument desc = failed to get connection: connecting failed: rados: ret=-13, Permission denied
I0929 17:13:10.313931       1 controller.go:1084] Final error received, removing PVC d373ab9b-1497-4718-b9f4-783d1ef5a5d4 from claims in progress
W0929 17:13:10.313954       1 controller.go:943] Retrying syncing claim "d373ab9b-1497-4718-b9f4-783d1ef5a5d4", failure 6
E0929 17:13:10.313983       1 controller.go:966] error syncing claim "d373ab9b-1497-4718-b9f4-783d1ef5a5d4": failed to provision volume with StorageClass "cephfs": rpc error: code = InvalidArgument desc = failed to get connection: connecting failed: rados: ret=-13, Permission denied
I0929 17:13:10.314060       1 event.go:282] Event(v1.ObjectReference{Kind:"PersistentVolumeClaim", Namespace:"default", Name:"csi-cephfs-pvc", UID:"d373ab9b-1497-4718-b9f4-783d1ef5a5d4", APIVersion:"v1", ResourceVersion:"43895367", FieldPath:""}): type: 'Warning' reason: 'ProvisioningFailed' failed to provision volume with StorageClass "cephfs": rpc error: code = InvalidArgument desc = failed to get connection: connecting failed: rados: ret=-13, Permission denied
```

**Comment by mixMugz** [comment]
Aditional. Mount from any node with command:

```
mount -t ceph IP:6789:/ /mnt/cephfs -o name=<client>,secret=<secret>
```
is success

**Comment by Rakshith-R** [comment]
hey @mixMugz , Can you please add

ceph version:
cephcsi version:
Logs from 
- csi-cephfsplugin-provisioner pod -> csi-cephfsplugin container  
- csi-cephfsplugin pod(on the nodes with problem) -> csi-cephfsplugin container

Make sure,
- ceph cluster is healthy
- verify ceph fs commands work with the particular id and keys used for cephcsi

And check whether https://docs.ceph.com/en/latest/security/CVE-2021-20288/ is the cause of the issue.

**Comment by mixMugz** [comment]
Thanks for reply. Tonight it was fixed just changing all images to v3.4.0. Was v3.2.0
